includes/class-wcmp-admin.php

```diff
@@ -53,7 +53,7 @@ public function order_list_shipment_options($order, $hide = true) {
             ?>
             <div class="wcmp_shipment_summary" <?php echo $style; ?>>
                 <?php $this->show_order_delivery_options($order); ?>
-                <a href="#" class="wcmp_show_shipment_summary"><span class="encircle wcmp_show_shipment_summary">i</span></a>
+                <a class="wcmp_show_shipment_summary"><span class="encircle wcmp_show_shipment_summary">i</span></a>
```

Comment:
Is `href="#"` niet nodig?

Reply:
Nope! Hierdoor kan ook event.preventDefault() weg uit de js, enige wat href doet is de cursor een pointer maken maar dat zit al in de css hier